src/components/Item/VisualizationItem/Visualization/IframePlugin.js

```diff
@@ -1,15 +1,24 @@
 import { useConfig } from '@dhis2/app-runtime'
 import { useD2 } from '@dhis2/app-runtime-adapter-d2'
+import { Layer, CenteredContent, CircularLoader } from '@dhis2/ui'
```

Comment:
```suggestion
import { CenteredContent, CircularLoader } from '@dhis2/ui'
```

This is giving me an eslint warning locally